### PR TITLE
Bug 1183870 - Set audio channel as none when audio channel is inactive

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bbce8a360b2391686c3244e19d29025e4ffbedaa

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6643f18c52a87feded6746569d49f44d03954326

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=238030bc10e77ab7e819ef7620015727b0552cf1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=37d4ac7d5e52c0573e2c4d1d081693b90605f158

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3b7457d0d1080d3ed8f8520372b4ae9dc78f499a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=698f2a3fca8f27b41afaa7eb9f8772c8c73a2b23

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=22690bf129c13b99d7f97e933b4a6936694a6806

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b3d1a477824ac5c784367e76f85f9325d1f82f5e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2c7148b3fd07ebc3854ad4b612bcdd3a2093591a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=269266cd2d6d9b9afd3631f28d84c407bf9a3c7b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=80101d4fd1363a1beb5c1055e54e8e411059dd4e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e7bb548fe7e5cce5af3ab78ea2800b20148be1ad

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7cddb2fe45fba33afd2353f19f6cea4cda8472ab

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ae760f31fe2835e6ecb4b3e51640dd70c2d35dfa

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bec15a39deabef9f1a5774d065a4dbfb241c65fa

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=101c5f411af5136529e2b4bcf515039bc726607f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cd0a6728b891594932c3c81940f1ebbd090d057a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0fec918c17b12eb9f8ad88117ece3d450662034c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e72b8842b33c08fcfc0a62da675b64cf39bed88b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ff9de15554ed9e6eb75570289efa93c3db833c90

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e6f34131c85b570904fdc0564ebbbe6886ad00e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=691626d7352e96f06f5ba82919e5356c1d59bc83

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e5b255978d5525a14b980d18e4667830854d75e6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7dcb4fdafdc823cadff9a7a2708119b8b64545d0

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=357a16e4e267d9d4c05798597c42b86dce5aa0b1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b21fb93c6cdd5e5bf4f71d15030888726349dec5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cb1ac1db315366d36bcfcfce426c0a4dbb1334fc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d39a68ebff39b6ab9fdca1e8485b933505ede8e9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=02c510bef905d5d7191d341695f3667100a10938

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=63a3134a5393b5cbd485859650169513e326a59b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0ef0e24fcc77bd3e5171a718112e93618b8d6264

File: apps/music/test/marionette/Player_test.js [hunk 1]
```diff
@@ -160,15 +160,22 @@ marionette('Music player tests', function() {
...
+      console.log('evanxd: before go home: getAttribute(hidden): ' +
+        statusbar.playingIndicator.getAttribute('hidden'));
 
       // switch to the homescreen
       var system = client.loader.getAppClass('system');
       system.goHome();
+      console.log('evanxd: going hame: getAttribute(hidden): ' +
+        statusbar.playingIndicator.getAttribute('hidden'));
       client.waitFor(function() {
         return client.findElement(system.Selector.activeHomescreenFrame)
```
Comment (lieza1989) [inline apps/music/test/marionette/Player_test.js]: Return client.findElement (sytem.Selector.ActiveHomescreenFrame) Displayed

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e2fc9f84501ae5dcdb7b8148df3f871240c0e09f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8a14827167cb2dbfdf74a049d33bf2d8d9ea511f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b4ee2788a2bd04826626fe022c9a949c7fcf4c2d

**Pull request merged**